cli/commands.go

```diff
@@ -125,12 +134,8 @@ func CreateCluster(c *cli.Context) error {
 		c.Bool("auto-restart"),
 	)
 	if err != nil {
-		log.Printf("ERROR: failed to create cluster\n%+v", err)
```

Comment:
Only a single comment here... we're not telling the user anymore that we failed to create the cluster? :thinking:

Reply:
The 'err' will be printed after the return.  This is actually a better log sequence -- Failed error log appears last, after the logs on cluster deletion, rather appear in the middle, before the logs on cluster deletion.